raster/src/main/scala/geotrellis/raster/ArrayMultiBandTileSubsetMethod.scala

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2016 Azavea.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package geotrellis.raster
+
+
+trait ArrayMultiBandTileSubsetMethod[T <: ArrayMultiBandTile] extends MethodExtensions[T] {
+
+  def subset(m: Map[Int, Int]): ArrayMultiBandTile = {
```

Comment:
Why should this be a map? I was thinking it would be a sequence of ints, with overloads to support passing values in directly:

```scala
def subset(bands: Seq[Int]): ArrayMultiBandTile = ???
def subset(bands: Int*)(implicit d: DummyImplicit): ArrayMultiBandTile = ???
```

You might need the dummy implicit to help resolve the overloads (scalac will tell you if it can't because signatures have the same type after type erasure).